src/process.c

```diff
@@ -61,7 +61,7 @@
 #include "netlink.h"
 
 static GMainLoop* main_loop = NULL;
-static GMainLoop* hook_loop = NULL;
+private GMainLoop* hook_loop = NULL;
```

Comment:
heh, OK, had to go look up where you got 'private' from ;-) - had shudders of C++ - but see that it comes from common.h and changes meaning depending on UNIT_TESTING :-)